src/popup/modules/UserInterface.js

```diff
@@ -591,60 +629,102 @@ export function init() {
+                } else {
+                    // regenerate QR code
+                    setNewQrCodeSize(qrLastSize, true);
+                }
             }
-        }
 
-        // also set height of text (also to prevent display errors) when remember is enabled
-        if (qrCodeSize.sizeType === "remember" && qrCodeSize.hasOwnProperty("sizeText")) {
-            console.info("restore qr code text size:", qrCodeSize.sizeText);
-            // is saved as CSS string already
-            // height is NOT (anymore) restored, as this may cause display errors (likely due to different content-box settings) and the height does not matter, anyway
-            qrCodeText.style.width = qrCodeSize.sizeText.width;
-
-            // detect too small size
-            const minimalSize = qrCodeSize.size + parseInt(qrCodeSize.sizeText.height, 10);
-            if (window.innerHeight < minimalSize) {
-                console.error("too small size", window.innerHeight, "should be at least: ", minimalSize);
+            // also set height of text (also to prevent display errors) when remember is enabled
+            if (
+                qrCodeSize.sizeType === "remember" &&
+                qrCodeSize.hasOwnProperty("sizeText")
+            ) {
+                console.info("restore qr code text size:", qrCodeSize.sizeText);
+                // is saved as CSS string already
+                // height is NOT (anymore) restored, as this may cause display errors (likely due to different content-box settings) and the height does not matter, anyway
+                qrCodeText.style.width = qrCodeSize.sizeText.width;
+
+                // detect too small size
+                const minimalSize =
+                    qrCodeSize.size + parseInt(qrCodeSize.sizeText.height, 10);
+                if (window.innerHeight < minimalSize) {
+                    console.error(
+                        "too small size",
+                        window.innerHeight,
+                        "should be at least: ",
+                        minimalSize
+                    );
+                }
             }
         }
-    });
+    );
 
     const initQrTypespecificSettings = createContextMenu();
 
     // return Promise chain
-    return Promise.all([applyingMonospaceFont, applyingQrSize, applyingQrColor, initQrTypespecificSettings]);
+    return Promise.all([
+        applyingMonospaceFont,
+        applyingQrSize,
+        applyingQrColor,
+        initQrTypespecificSettings,
+    ]);
 }
+
+/**
+ * Listens to Alt + plus/minus keypresses to alter 
+ * QR code size.
+ */
+document.addEventListener('keydown', function(event) {
+    if ((event.key == '=' || event.key == '+') && event.altKey && qrLastSize < 440) {
+        setNewQrCodeSize(qrLastSize + 30, true);
+
+        localStorage.setItem("lastSize", qrLastSize);
+    }
+    if (event.key == '-' && event.altKey && qrLastSize > 50) {
+        setNewQrCodeSize(qrLastSize - 30, true);
```

Comment:
Ah and could you extract the `30` into a `const KEYBOARD_ZOOM_SIZE = 30` constant or so? Magic numbers are not a good practise in source code that should be readable… :upside_down_face:

Reply:
Same for `50` which I assume is something as `MINIMAL_ZOOM_SIZE` or so?

Just place the constants at the top, there should be some, already. :wink: